include/deal.II/lac/petsc_sparse_matrix.h

```diff
@@ -397,6 +403,12 @@ namespace PETScWrappers
        */
       SparseMatrix();
 
+      /**
+       * Initialize a SparseMatrix from a PETSc Mat object. Note that we do not
+       * copy the matrix.
+       */
+      explicit SparseMatrix(const Mat &);
```

Comment:
And here

Reply:
https://github.com/dealii/dealii/pull/14481/commits/d76f474c345dcee0b12eb28f64fe1ac890b9665f